wolfssh/settings.h

```diff
@@ -72,6 +72,15 @@ extern "C" {
     #error only SCP server side supported
 #endif
 
+/* Detect if ECC needs RNG */
+#if !defined(HAVE_WC_ECC_SET_RNG) && \
```

Comment:
For reference this normally comes from the configure.ac here https://github.com/wolfSSL/wolfssh/blob/master/configure.ac#L126, however when building without configure this causes an issue with all ECC signing.